src/comprehensions.py

```diff
@@ -10,14 +10,15 @@
 
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [x for x in range(6)]
```

Comment:
format for this would be just a little different, try "y = [i for i in range(1, 6)]" otherwise 0 may be included in the printed array